compiler/cli/bin/kotlinc.bat

```diff
@@ -103,5 +106,8 @@ rem Needs to be executed in the EnableDelayedExpansion mode.
   )
 goto :eof
 
+:error
+set ERRORLEVEL=1
+
 :end
-endlocal
```

Comment:
I removed the `endlocal` for following reasons.

1. `endlocal` will not be executed after `exit`(which is the current situation before I made this pull request), and if I put it before `exit /b %ERRORLEVEL%`, the %ERRORLEVEL%` will always  be 0.
2. according to [official document](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/endlocal), there is an implicit `endlocal` command at the end of a batch file.